testdata/baselines/reference/submoduleAccepted/conformance/jsdocParseStarEquals.errors.txt.diff

```diff
@@ -3,19 +3,20 @@
 @@= skipped -0, +-1 lines =@@
 -<no content>
 @@= skipped --1, +1 lines =@@
++a.js(1,5): error TS1047: A rest parameter cannot be optional.
```

Comment:
Is this an intentional change in behavior? Previously a postfix `=` meant optional-if-able-but-`| undefined`-otherwise.